### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ghost**
#648

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ represented by a folder (e.g., [`files/en-us/web/javascript`](files/en-us/web/ja
...
-* **A document's `index.html` starts with "front-matter"** -- Each document's `index.html` file must begin with some YAML called front-matter that defines some important information about the document: `title`, `slug`, and [`tags`](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Howto/Tag) (if any). Here's an example that shows the front-matter from the [JavaScript landing page](files/en-us/web/javascript/index.html):
+* **A document's `index.html` starts with "front-matter"** -- Each document's `index.html` file must begin with some [YAML](https://en.m.wikipedia.org/wiki/YAML) called front-matter that defines some important information about the document: `title`, `slug`, and [`tags`](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Howto/Tag) (if any). Here's an example that shows the front-matter from the [JavaScript landing page](files/en-us/web/javascript/index.html):
```
Comment (nschonni) [inline README.md:59]: Any reason to do the "mobile" wikipedia (with the `.m.`) vs the desktop site?

Comment (mattiapontonio) [reply README.md:59]: I made the change using the responsive web version of GitHub. Wikipedia therefore obviously has a special domain if you visit it from mobile, which can be avoided by making a responsive site.

If you want you can suggest a change.

On the other hand, it is absurd how green there is for a URL.

Comment (mattiapontonio) [reply README.md:59]: No preference for me to use the desktop or mobile URL.

Comment (nschonni) [reply README.md:59]:
```suggestion
* **A document's `index.html` starts with "front-matter"** -- Each document's `index.html` file must begin with some [YAML](https://en.wikipedia.org/wiki/YAML) called front-matter that defines some important information about the document: `title`, `slug`, and [`tags`](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Howto/Tag) (if any). Here's an example that shows the front-matter from the [JavaScript landing page](files/en-us/web/javascript/index.html):
```

**Review by chrisdavidmills — APPROVED** [review]
LGTM; thanks @mattiapontonio !

**Pull request merged**